src/main/java/datawave/data/normalizer/LcNoDiacriticsNormalizer.java

```diff
@@ -21,6 +21,7 @@
 public class LcNoDiacriticsNormalizer extends AbstractNormalizer<String> {
     private static final long serialVersionUID = -7922074256473963293L;
     private static final Pattern diacriticals = Pattern.compile("\\p{InCombiningDiacriticalMarks}");
+    private static final Pattern capitals = Pattern.compile("[A-Z]");
```

Comment:
Not sure this is actually correct as we could use capital letters as part of a boundary matcher like \B  (a non-word boundary) or pattern class like \p{Digit}.
Also does not handle other upper case matching character classes like \p{Upper}.
The JavaRegexAnalyzer may be able to handle this better.
In the end, we may not want this one to implement the method because users already know that a regex is case insensitive.

Reply:
1. yeah, I just added this as a placeholder for now, wasn't sure there'd be other characters/patterns
2. yesterday you told me explicitly you wanted this in LcNoDiacritics for case sensitivity.  What changed your mind?

Reply:
Admittedly I go back and forth on whether we should allow regexes case and diacritic sensitive.  So this is my fault.  It will require user education if we start making this case and diacritic dependent.  We could look through the history of queries to determine whether anybody is actually doing that or not.  In anycase, if we decide to go forward with an implementation here, we need a more robust one.

Reply:
I would love to move forward with accepting this PR (without and implementation here) and add a separate ticket for that.  We need to fix what we have deployed.

Reply:
This can be removed.